operator/v1/types.go

```diff
@@ -33,14 +33,41 @@ type ClusterManagerSpec struct {
 
 // ClusterManagerStatus represents the current status of the registration and work distribution controllers running on the hub.
 type ClusterManagerStatus struct {
+	// ObservedGeneration is the last generation change you've dealt with
+	// +optional
+	ObservedGeneration int64 `json:"observedGeneration,omitempty" protobuf:"bytes,1,opt,name=observedGeneration"`
+
 	// Conditions contain the different condition statuses for this ClusterManager.
 	// Valid condition types are:
 	// Applied: components in hub are applied.
 	// Available: components in hub are available and ready to serve.
 	// Progressing: components in hub are in a transitioning state.
 	// Degraded: components in hub do not match the desired configuration and only provide
 	// degraded service.
-	Conditions []StatusCondition `json:"conditions" protobuf:"bytes,1,opt,name=conditions"`
+	Conditions []StatusCondition `json:"conditions" protobuf:"bytes,2,opt,name=conditions"`
```

Comment:
you cannot change the proto tag serialization order (that int after bytes)

Reply:
We should just remove proto for now.